app/components/Gallery.js

```diff
@@ -41,13 +42,26 @@ export default class Gallery extends Component {
     return Math.floor(Math.random()*(max-min+1)+min);
   }
 
+  settingsFromBreakpoint() {
+    if (this.props.breakpoint === 'small') {
+      return {
+        variableWidth: this.randomNumberFromRange(95, 100),
+        nudgeDistance: '10%'
+      } 
+    }
+    return {
+      variableWidth: this.randomNumberFromRange(50, 90),
+      nudgeDistance: '20%'
+    } 
+  }
+
```

Comment:
I think this is cool!  ideally we'd want every to do with styles inside an Atomic / Radon thing, but I'm not sure it's possible here, i'd need to take a closer look